sale_order_line_packaging_qty/models/sale_order_line.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class SaleOrderLine(models.Model):
+
+    _inherit = "sale.order.line"
+
+    product_packaging_qty = fields.Float(
+        string="Package quantity",
+        compute="_compute_product_packaging_qty",
+        inverse="_inverse_product_packaging_qty",
+        digits="Product Unit of Measure",
+    )
+
+    @api.depends(
+        "product_uom_qty", "product_uom", "product_packaging", "product_packaging.qty"
+    )
+    def _compute_product_packaging_qty(self):
+        for sol in self:
+            if (
+                not sol.product_packaging
+                or sol.product_uom_qty == 0
+                or sol.product_packaging.qty == 0
+            ):
+                sol.product_packaging_qty = 0
+                continue
+            # Consider uom
+            if sol.product_id.uom_id != sol.product_uom:
+                product_qty = sol.product_uom._compute_quantity(
+                    sol.product_uom_qty, sol.product_id.uom_id
+                )
+            else:
+                product_qty = sol.product_uom_qty
+            sol.product_packaging_qty = product_qty / sol.product_packaging.qty
+
+    def _prepare_product_packaging_qty_values(self):
+        return {
+            "product_uom_qty": self.product_packaging.qty * self.product_packaging_qty,
+            "product_uom": self.product_packaging.product_uom_id.id,
+        }
+
+    def _inverse_product_packaging_qty(self):
+        for sol in self:
+            if not sol.product_packaging:
+                raise UserError(
+                    _(
+                        "You must define a package before setting a quantity "
+                        "of said package."
+                    )
+                )
+            if sol.product_packaging.qty == 0:
```

Comment:
```suggestion
            if sol.product_packaging_qty == 0:
```
and you may need to check it every where